plans/benchmarks/benchmarks.go

```diff
@@ -260,15 +208,16 @@ func SubtreeBench(runenv *runtime.RunEnv) error {
 
 		for _, tst := range tests {
 			for i := 1; i <= iterations; i++ {
-				t := prometheus.NewTimer(tst.Summary)
+				t := time.Now()
 				_, err = client.Publish(ctx, tst.Topic, tst.Data)
 				if err != nil {
 					return err
 				}
-				t.ObserveDuration()
+				tst.Timer.UpdateSince(t)
```

Comment:
It will be interesting to graph this alongside the `ResettingTimer` once we introduce it.

Reply:
Yup, totally. This timer is kinda useless.

Reply:
linking, because the web and GitHub allow us to: https://github.com/testground/testground/issues/937